### ISSUE TITLE
Popup Function generate blue lines

### ISSUE THREAD

**Issue by mahad1** [driftyco/ionic#1169]
Hello everyone
First of all, I thank the ionic team for this awsome framework.

I have a problem with the popup function. I am using Android 4.2.2 on Samsung S1 (I9000). 
After popup is showed and clicked on the green button 

http://forum.ionicframework.com/t/popup-function-generate-blue-lines/3177
The app react very strange. It shows me blue line like on the picture below (On Text: Paracetamol Hans..)

http://forum.ionicframework.com/t/popup-function-generate-blue-lines/3177
Basically it has a problem with the popup function. Because the Touch Funktion is also working not fine after a popup is shown.

Could anyone help me out of this. Thanks in advance.
Regards
Mahad
